src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -273,7 +273,7 @@ def load_pytorch_state_dict_in_tf2_model(
 
         array = apply_transpose(transpose, pt_state_dict[name], symbolic_weight.shape)
 
-        tf_loaded_numel += array.size
+        tf_loaded_numel += tensor_size(array)
```

Comment:
This should have been done in the previous PR refactoring all utils in a framework-agnostic way, but did not notice this was here :man_shrugging: